tyssue/behaviors/sheet_events.py

```diff
@@ -19,6 +22,8 @@ def __init__(self, sheet, model, geom):
         self.sheet = sheet
         self.model = model
         self.geom = geom
+        self.current_deque = deque()
```

Comment:
I'm going to rename these from `current_deque`, `next_deque` to just`current`, `next`